### PR TITLE
Update pictures.en.md

### PR DESCRIPTION
**Pull request by rosaguilar**
Add information about capturing and storing EXIF info of pictures.
Cheers from SA

### CODE REVIEW

File: documentation/how-to/pictures.en.md [hunk 1]
```diff
@@ -77,6 +77,23 @@ To enable geotagging in case your native OS camera does not support this functio
     deactivated
     on your mobile device
 3.  Completed! QField's own QML-based camera will now be utilized while taking pictures
+   
+## Saving Geotags (EXIF) automatically
+:material-monitor: Desktop preparation
+
+Sometimes you might be interested in storing automatically Geotags such as the latitude, longitude, orientation, etc. This information is also known as EXIF tags.
```
Comment (lucienicolier) [inline documentation/how-to/pictures.en.md:84]:
```suggestion
Sometimes you might be interested in automatically storing Geotags such as the latitude, longitude, orientation, etc. This information is also known as EXIF tags.
```

File: documentation/how-to/pictures.en.md [hunk 2]
```diff
@@ -77,6 +77,23 @@ To enable geotagging in case your native OS camera does not support this functio
...
+
+To store the EXIF information, follow these steps:
+
+1.  Add an attribute per EXIF tag in the table that contains the pictures.
+2.  In the pictures form, configure the default value of each attribute to the corresponding
+   *EXIF* expression [See QGIS EXIF function](https://docs.qgis.org/3.28/en/docs/user_manual/expressions/functions_list.html?highlight=exif#exif),
```
Comment (lucienicolier) [inline documentation/how-to/pictures.en.md:90]:
```suggestion
   *EXIF* expression !![See QGIS EXIF function](https://docs.qgis.org/3.28/en/docs/user_manual/expressions/functions_list.html?highlight=exif#exif),
```

File: documentation/how-to/pictures.en.md [hunk 3]
```diff
@@ -77,6 +77,23 @@ To enable geotagging in case your native OS camera does not support this functio
...
+    and make sure *Apply on update* is activated.
+4.  The EXIF tags that QField can capture are listed in the QGIS documentation (link above).
+    However, this list might slightly varies depending on the mobile characteristics.
+5.  Capturing EXIF tags requires to access the full physical path of the picture. Be sure of reflecting this in the QGIS expression.
```
Comment (lucienicolier) [inline documentation/how-to/pictures.en.md:94]:
```suggestion
5.  Capturing EXIF tags requires accessing the full physical path of the picture. Be sure to reflect this in the QGIS expression.
```

**Review by lucienicolier — COMMENTED** [review]
@rosaguilar Thank you so much for your contribution.
I have added a few suggestions.

File: documentation/how-to/pictures.en.md [hunk 4]
```diff
@@ -77,6 +77,23 @@ To enable geotagging in case your native OS camera does not support this functio
     deactivated
     on your mobile device
 3.  Completed! QField's own QML-based camera will now be utilized while taking pictures
+   
+## Saving Geotags (EXIF) automatically
```
Comment (lucienicolier) [inline documentation/how-to/pictures.en.md:81]:
```suggestion
## Fetching Geotags (EXIF) from the image file into the attribute table
```

**Review by lucienicolier — COMMENTED** [review]
@rosaguilar I just changed the title to make it clearer

**Comment by m-kuhn** [comment]
This PR has already been merged 5 hours ago.
Can someone create a new pull request (from master) with a fix?

**Pull request merged**